watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -382,16 +382,24 @@ def build(self):
             else:
                 self.neutral_set.add(j)
 
+        if len(self.neutral_set)>0 and not len(self.ion_set):
+            _log.warning("Since the charge argument was not provided for any solutes while instantiating the MCAS property model, all solutes are assumed to be neutral and have no charge. If any ions exist in solute_list, make sure to provide charge data to avoid erroneous property calculations.")
         # reference
         # Todo: enter any relevant references
 
+        # Check for molecular weight data
+        if not len(self.config.mw_data):
+            raise ConfigurationError("The mw_data argument was not provided while instantiating the MCAS property model. Provide a dictionary with solute names and associated molecular weights as keys and values, respectively.")
+      
         # TODO: consider turning parameters into variables for future param estimation
+        mw_temp = {"H2O": 18e-3}
+        mw_temp.update(self.config.mw_data)
```

Comment:
Adding H2O here so that the user doesn't have to anymore. However, our older code where we had to add MW data explicitly for H2O would still work (even though it'd be unnecessary.

Reply:
only critique here, is this will replace a user-provided value. I wounder if instead, we should check if the user provided a value for H2O in first place, and only update if the have not.

Reply:
Ah- that was also my concern, so I figured writing it this way (putting our default for H2O first, then updating the dictionary with whatever the user provided) would prevent overwriting what the user entered for "H2O".

As a sanity check, I just added one last test to verify that the user's provided value for H2O mw data wouldn't be replaced.

Reply:
https://github.com/watertap-org/watertap/blob/be1aeacb6cfc6b400500691d42d4cfbf4ab50c3e/watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py#L2229-L2243

Reply:
oh, oops. I misread the code. I thought you were updating the user-provided config with a default value. my bad. all good then.